spec/models/service_spec.rb

```diff
@@ -7,7 +7,9 @@
         'grogu' => 'grogu',
         'Darth Vader' => 'darth-vader',
         'emperor-palpatine' => 'emperor-palpatine',
-        'princess Leia' => 'princess-leia'
+        'princess Leia' => 'princess-leia',
+        "Darth Vader Children's" => 'darth-vader-children-s',
+        'Darth Vader Children’s' => 'darth-vader-children-s'
```

Comment:
```suggestion
        "Darth Vader Children's" => 'darth-vader-childrens',
        'Darth Vader Children’s' => 'darth-vader-childrens'
```
should the urls look like this?
So it is similar to the `claim-for-costs-of-a-childs-funeral.form.service.justice.gov.uk` url.